### PR TITLE
Add EFI constants according to spec

### PR DESCRIPTION
**Pull request by WesterWest**
The code currently uses only string based constants when parsing ELF files

This PR sets to solve this. It should also not be susceptible to bugs due to issues in the zig compiler about packed structs. I've tried to make sure, that no fields cross the alignment boundaries anywhere.

### CODE REVIEW

**Comment by artix-boi** [comment]
Okay, i think i got FileHeader.zig SectionHeader.zig and ProgramHeader.zig it work with arch and elf.zig. 
ig we can merge this?

File: LICENSE [hunk 1]
```diff
@@ -1,25 +1,21 @@
+The MIT License (Expat)
 
-This is free and unencumbered software released into the public domain.
+Copyright (c) 2015-2021, Zig contributors
```
Comment (WesterWest) [inline LICENSE:3]: This should not be a "Zig contributors", the copyright as zig contributors should only appear before code taken from the zig std

**Review by WesterWest — COMMENTED** [review]
Otherwise it look good!

File: LICENSE (hunk 1 above)
Comment (artix-boi) [reply LICENSE]: fixed

**Pull request merged**